src/components/AvatarWithDisplayName.tsx

```diff
@@ -133,16 +123,16 @@ function AvatarWithDisplayName(props) {
                             displayNamesWithTooltips={displayNamesWithTooltips}
                             tooltipEnabled
                             numberOfLines={1}
-                            textStyles={[props.isAnonymous ? styles.headerAnonymousFooter : styles.headerText, styles.pre]}
-                            shouldUseFullTitle={isMoneyRequestOrReport || props.isAnonymous}
+                            textStyles={[isAnonymous ? styles.headerAnonymousFooter : styles.headerText, styles.pre]}
+                            shouldUseFullTitle={isMoneyRequestOrReport || isAnonymous}
                         />
-                        {!_.isEmpty(parentNavigationSubtitleData) && (
+                        {Object.keys(parentNavigationSubtitleData).length > 0 && (
                             <ParentNavigationSubtitle
                                 parentNavigationSubtitleData={parentNavigationSubtitleData}
-                                parentReportID={props.report.parentReportID}
+                                parentReportID={report?.parentReportID}
                             />
                         )}
-                        {!_.isEmpty(subtitle) && (
+                        {!!subtitle && (
```

Comment:
```suggestion
                        {subtitle && (
```

this works?

Reply:
@fabioh8010  If `subtitle` is an empty string it will cause a console error on the web.
![image](https://github.com/Expensify/App/assets/23176449/8ba2a535-b842-44ab-85ee-18c05098e6eb)